app/views/directives/traffic-table.html

```diff
@@ -0,0 +1,84 @@
+<!-- Table combining routes and ports for a service -->
+<table class="table table-bordered table-hover table-mobile" style="{{ hasFooter ? 'margin-bottom:5px;' : ''}}">
```

Comment:
when i saw hasFooter i expected the footer to actually be part of the directive, not to just change the styles of the table.  i would either make the footer part of the directive, or you can use negative margins on the footer.